### PR TITLE
Lucas/refactor drawing commands

### PR DESCRIPTION
**Pull request by lgrossi**
First primitive attempt of joining deletion and drawing in the same system flow

### CODE REVIEW

File: ryot_compass/src/bevy_compass/drawing/delete.rs [hunk 1]
```diff
@@ -19,36 +19,45 @@ pub fn erase_on_click() -> SystemConfigs {
 
 /// A function that listens to the right mouse button and deletes the content of the tile under the cursor.
 /// It always delete the topmost content of the tile, following the Z-ordering.
```
Comment (luan) [inline ryot_compass/src/bevy_compass/drawing/delete.rs:21]: this docstring is out of date

File: ryot_compass/src/bevy_compass/camera.rs [hunk 2]
```diff
@@ -67,37 +68,37 @@ pub struct Cursor {
...
+    pub input_type: DrawingInputType,
     pub enabled: bool,
     pub brush_index: usize,
 }
 
 #[derive(Eq, PartialEq, Clone, Copy, Reflect)]
-pub enum DrawingMode {
+pub enum DrawingInputType {
     Click(i32),
     TwoClicks(Option<TilePosition>),
```
Comment (luan) [inline ryot_compass/src/bevy_compass/camera.rs:79]: Is this "Drag" now?

File: ryot_compass/src/bevy_compass/drawing/draw.rs [hunk 3]
```diff
@@ -34,9 +33,9 @@ fn draw_to_tile<C: ContentAssets, F: ReadOnlyWorldQuery>(
     };
 
     for (AppearanceDescriptor { group, id, .. }, tile_pos, cursor) in &cursor_query {
-        if !cursor.drawing_state.enabled {
-            continue;
-        }
+        // if !cursor.drawing_state.enabled {
+        //     continue;
+        // }
```
Comment (luan) [inline ryot_compass/src/bevy_compass/drawing/draw.rs:38]: oi

File: ryot_compass/src/bevy_compass/drawing/delete.rs (hunk 1 above)
Comment (lgrossi) [reply ryot_compass/src/bevy_compass/drawing/delete.rs:21]: Good point I need to review the whole documentation, I'll do it

**Pull request merged**